simpeg/optimization.py

```diff
@@ -248,6 +237,22 @@ class IterationPrinters(object):
         "format": "%3d",
     }
 
+    iteration_CG_rel_residual = {
+        "title": "CG |Ax-b|/|b|",
+        "value": lambda M: M.cg_rel_resid,
+        "width": 15,
+        "format": "%1.2e",
+        # "format": lambda v: f"{v:1.2e}",
+    }
+
+    iteration_CG_abs_residual = {
+        "title": "CG |Ax-b|",
+        "value": lambda M: M.cg_abs_resid,
+        "width": 11,
+        "format": "%1.2e",
+        # "format": lambda v: f"{v:1.2e}",
+    }
+
```

Comment:
Open to suggestions on the title of these two writers... some thoughts of mine:
* `CG rel resid`, `CG abs resid`
* `CG residual_r`, `CG residual_a`
* ... ?

Reply:
What about `CG residual (abs)` and `CG residual (rel)`?